src/common/benches/bitmap.rs

```diff
@@ -28,5 +29,33 @@ fn bench_bitmap(c: &mut Criterion) {
     c.bench_function("not", |b| b.iter(|| !&x));
 }
 
-criterion_group!(benches, bench_bitmap);
+/// Bench ~10 million records
+fn bench_bitmap_iter(c: &mut Criterion) {
+    const CHUNK_SIZE: usize = 1024;
+    const N_CHUNKS: usize = 10_000;
+    fn make_iterators(bitmaps: &[Bitmap]) -> Vec<BitmapIter<'_>> {
+        bitmaps.iter().map(|bitmap| bitmap.iter()).collect_vec()
+    }
+    fn bench_bitmap_iter(bench_id: &str, bitmap: Bitmap, c: &mut Criterion) {
+        let bitmaps = vec![bitmap; N_CHUNKS];
+        let iters = || make_iterators(&bitmaps);
+        let result = || vec![true; CHUNK_SIZE];
+        c.bench_function(bench_id, |b| {
+            b.iter(|| {
+                let mut result = result();
```

Comment:
Is this step also counted into the total time?
If so, can https://bheisler.github.io/criterion.rs/criterion/struct.Bencher.html#method.iter_batched help

Reply:
I guess an array outside would be okay.
```rust
let mut result = vec![true; CHUNK_SIZE];
c.bench_function(bench_id, |b| {
    b.iter(|| {
        ...
    }
}
```